README.md

```diff
@@ -339,22 +339,30 @@ admin@ncs# devtools true
 
 Then do the configuration that you want to do through maagic API. Instead of commit use the `show configuration | display maagic` command. To continue with the authgroup example you would do:
 ```
+config
 devices authgroups group test default-map same-user same-pass
+top
 show configuration | display maagic 
 ```
 Output:
 ```
 admin@ncs(config)# devices authgroups group test default-map same-user same-pass
+admin@ncs(config)# top
 admin@ncs(config)# show configuration | display maagic 
 root = ncs.maagic.get_root(t)
 root.ncs__devices.authgroups.group.create('default')
-root.ncs__devices.authgroups.group['test'].default-map.create()
-root.ncs__devices.authgroups.group['test'].default-map.same-user.create()
-root.ncs__devices.authgroups.group['test'].default-map.same-pass.create()
+root.ncs__devices.authgroups.group['default'].default-map.create()
+root.ncs__devices.authgroups.group['default'].default-map.same-user.create()
+root.ncs__devices.authgroups.group['default'].default-map.same-pass.create()
 ```
 
 Note that there is a minor bug so some dashes (`-`) need to be replaced with underscores (`_`).
 
+Exit NSO CLI:
+```
+admin@ncs# exit
```

Comment:
we need double exit, since we where on config mode. Not sure if there is a way to exit NSO from config mode.

```bash
admin@ncs(config)# show configuration | display maagic 
root = ncs.maagic.get_root(t)
admin@ncs(config)# 
admin@ncs(config)# exit
admin@ncs# exit
developer:src > 
```